### DISCUSSION TITLE
-clip behavior

### DISCUSSION THREAD

**Discussion by cedeymard9** [ImageMagick/ImageMagick#4620] [Help] [answered]
Hi all, 

I try to transform different images sources to a png transparent. With my tests on multiple servers, I have different results with the same command. I reduce my command to the basic to identify which parameters has a different behavior : 
convert image.eps -clip out.png
With one server on IM 6.9.9-26 and GS 9.22, I have the result I wanted : image.png
With another server on IM 6.9.10-23 and GS 9.27, I have a different result : imageDifferent.png
![image](https://user-images.githubusercontent.com/66676689/146928648-7ef179ea-f1ee-4e25-bb90-8da1bd5449ac.png)
![imageDifferent](https://user-images.githubusercontent.com/66676689/146928709-181a1ac0-cc41-4d9e-b10f-e438bbd23620.png)

It works with many images, but I have this one that is different between servers.
I didn't find explicite information for me in -verbose mode

My final goal is to have a transparent png at the end, and for that I launch two command : 
1 / convert imageSource.eps -alpha set -channel alpha -clip -negate temp.png
2 / convert temp.png -channel alpha -negate out.png

This is the link to original image if needed : 
[image.eps.zip](https://github.com/ImageMagick/ImageMagick/files/7755217/image.eps.zip)

**Comment by dlemstra** [comment, marked as answer]
I am unable to reproduce this issue with the most recent version of ImageMagick 6.

**Reply by cedeymard9** [reply]
Thanks for your answer ! 
I will try to install last version 6 and test again.
